masterlist.yaml

```diff
@@ -524,6 +524,22 @@ plugins:
       - 'Graphic Herbalism TR.ESP'
       - 'Graphic Herbalism TR Extra.ESP'
 
-#Any dynamic patches (not sure if multipatch is the  only one)
+  #Any dynamic patches
+  - name: 'Merged Objects.esp'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/46870/' ]
+    group: *dynamicPatchGroup
+    msg:
+      - type: say
+        content:
+          - lang: en
+            text: 'Regenerate after adding or removing mods that change or add in-game items. Use [TES3Merge](https://www.nexusmods.com/morrowind/mods/46870/)'
+
   - name: 'multipatch.esp'
     group: *dynamicPatchGroup
+    after:
+      - 'Merged Objects.esp'
+    msg:
+      - type: say
+        content:
+          - lang: en
+            text: 'Regenerate after adding or removing mods or generating [Merged Objects.esp](https://www.nexusmods.com/morrowind/mods/46870/)'
```

Comment:
Missing newline at the end of file.

![image](https://user-images.githubusercontent.com/1944639/138538111-3c5b5802-e408-4941-b8c4-a756e36c2f24.png)

```suggestion
            text: 'Regenerate after adding or removing mods or generating [Merged Objects.esp](https://www.nexusmods.com/morrowind/mods/46870/)'

```